Project.toml

```
@@ -68,11 +69,14 @@ StatsFuns = "0.8, 0.9, 1"
 Tracker = "0.2.3"
 julia = "1.7"
 
-[weakdeps]
-Optim = "429524aa-4258-5aef-a3af-852621145aeb"
-
 [extensions]
+DynamicHMCExt = "DynamicHMC"
```

Following the common convention, can you rename the extension to
```suggestion
TuringDynamicHMCExt = "DynamicHMC"
```
?